src/settings/DepartmentsSettings.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {
+  FormattedMessage,
+  useIntl,
+} from 'react-intl';
+
+import { NoValue } from '@folio/stripes/components';
+import { ControlledVocab } from '@folio/stripes/smart-components';
+import { useStripes } from '@folio/stripes/core';
+
+const DepartmentsSettings = () => {
+  const intl = useIntl();
+  const stripes = useStripes();
+  const ConnectedControlledVocab = stripes.connect(ControlledVocab);
```

Comment:
`ConnectedControlledVocab` will be mounted from scratch every time when `DepartmentsSettings` renders. To avoid this, a would suggest wrapping `stripes.connect(ControlledVocab);` with `useMemo`:
```js
const ConnectedControlledVocab = useMemo(() => stripes.connect(ControlledVocab), [stripes]);
```